express/server.js

```diff
@@ -14,6 +14,20 @@
 
 'use strict';
 
+const makeRelease = function() {
```

Comment:
I suggest we name this `determineReleaseVersion` or something similar. When I read `makeRelease` my brain thinks that there's a release/deploy happening as a result of this function, whereas what it's really doing is determining the value of the release version.

Reply:
I renamed it.